src/main/java/org/opentripplanner/api/mapping/LegMapper.java

```diff
@@ -80,6 +80,9 @@ public ApiLeg mapLeg(Leg domain, Calendar arrivalTimeFromPlace, Calendar departu
             api.tripShortName = trip.getTripShortName();
             api.tripBlockId = trip.getBlockId();
         }
+        else if (domain.pathway) {
+            api.route = domain.edgeIdentifier.toString();
```

Comment:
We avoid using `toString` anywhere in the mapping. The internal model objects uses the `toString` for logging and debugging ONLY. Here you should use the: `FeedScopedIdMapper.mapToApi(..)`, also we want to support different id implementation in the future, and this code would fail silently if we do that.